### PR TITLE
Thoracic Site Setup

### PR DESCRIPTION
**Pull request by Shinsina**
Initial-Setup

### CODE REVIEW

**Comment by Shinsina** [comment]
[New site: Thoracic](https://trello.com/c/kDnDdp3n/325-new-site-thoracic)

File: docker-compose.yml [hunk 1]
```diff
@@ -370,6 +370,22 @@ services:
...
+    working_dir: /root/sites/thoracicam.com
+    environment:
+      <<: *env
+      <<: *env-tauron
+      PORT: 80
+      EXPOSED_PORT: 9928
+      LIVERELOAD_PORT: 19928
+      TENANT_KEY: ascend_hh
+      SITE_ID: 62f156eeb42f9308288b45e8
+    ports:
```
Comment (solocommand) [inline docker-compose.yml:385]: All BaseCMS website service definitions should now use the `hostname` service key, and include the `HOST` environment variable, to ensure a consistently unique cookie domain across environments. This ensure both that the IdentityX, Omeda, and P1Events contexts (where applicable) will be correct when switching between sites.

```suggestion
      SITE_ID: 62f156eeb42f9308288b45e8
      HOST: www-ascend-thoracic.dev.parameter1.com
    hostname: www-ascend-thoracic.dev.parameter1.com
    ports:
```

File: .github/workflows/deploy-staging.yml [hunk 2]
```diff
@@ -54,6 +54,7 @@ jobs:
           - aao.hns.org
           - ashp.org
           - shm.org
+          - thoracicam.com
```
Comment (solocommand) [inline .github/workflows/deploy-staging.yml:57]: May as well add to the production deploy so it doesn't get forgotten, even if the container infrastructure/routing hasn't been stood up yet.

**Review by solocommand — CHANGES_REQUESTED** [review]

**Review by solocommand — APPROVED** [review]

**Comment by brandonbk** [comment]
@Shinsina - closing this for now...it wasn't communicated that this site will not use the same theme as previous sites.  We can reopen and/or steal the appropriate pieces after next week's meeting.

**Pull request closed without merging**